install/helm/agones/values.yaml

```diff
@@ -124,6 +124,7 @@ agones:
     http:
       port: 443
       serviceType: LoadBalancer
+      loadBalancerIP: ""
```

Comment:
Can we add that in the [helm installation](https://github.com/googleforgames/agones/blob/master/site/content/en/docs/Installation/Install%20Agones/helm.md) documentation as well? 

Also, we should note that it only works if the k8s provider supports that.

Reply:
Done!